packages/server/README.md

```diff
@@ -1,21 +1,22 @@
 # Installing
 
-## Configuration
+## Requirements
 
-First copy `.env-sample` to `.env` and edit the values in there:
+- Docker Engine runs Joplin Server. See [Install Docker Engine](https://docs.docker.com/engine/install/) for steps to install Docker Engine for your operating system.
+- Docker Compose is required to store item contents (notes, tags, etc.) if PostgreSQL is not used. See [Install Docker Compose](https://docs.docker.com/compose/install/) for steps to install Docker Compose for your operating system.
 
-- `APP_BASE_URL`: This is the base public URL where the service will be running. For example, if you want it to run from `https://example.com/joplin`, this is what you should set the URL to. The base URL can include the port.
-- `APP_PORT`: The local port on which the Docker container will listen. You would typically map this port to 443 (TLS) with a reverse proxy.
+## Configure docker for joplin server
```

Comment:
I  know I've said the titles should be lowercase, but proper names should have an uppercase, so "Docker" and "Joplin Server". It's Title Like This We Don't Do Here. I have nothing against it but for consistency with the rest of the documentation we don't do that (everything uppercase is more a US thing).

Reply:
Thanks! Just submitted the updates.